model/Notification/Search.ts

```diff
@@ -0,0 +1,12 @@
+import { isly } from "isly"
+
+export interface Search {
+	unreadOnly?: boolean
+}
+
+export namespace Search {
+	export const type = isly.object<Search>({
+		unreadOnly: isly.boolean().optional(),
+	})
+	export const is = type.is
+}
```

Comment:
unused, also not present in the api

Reply:
Whoops, think I looked at the NotifcationSearchCritera. 
Removed